includes/wikia/services/WikiService.class.php

```diff
@@ -479,7 +479,7 @@ public function getMostLinkedPages( $wikiId = 0 ) {
 	public function getMostActiveAdmins($wikiId, $avatarSize) {
 		$edits = $ids = $lastRevision = [];
 		$admins = $this->getWikiAdmins($wikiId, $avatarSize);
-		$ids = array_map(function($item) { return $item['userId']; }, $admins);
+		$ids = array_map(function( Array $item ) { return isset( $item['userId'] ) ? $item['userId'] : 0; }, $admins);
```

Comment:
It shouldn't take place. But if it will happen, we will not fetch information about such "ghost user"